components/calendly/calendly.app.js

```diff
@@ -0,0 +1,55 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "calendly",
+  methods: {
+    async _getHeader() {
```

Comment:
Please don't declare functions async if not using `await` inside (marking functions returning promises is nice too but unnecessary)